explore-analyze/discover/try-esql.md

```diff
@@ -130,3 +130,62 @@ FROM kibana_sample_data_ecommerce
 :alt: ESQL query with a custom time field enabled
 :::
 
+### Create and edit lookup indices from LOOKUP JOINs commands [discover-esql-lookup-join]
+```{applies_to}
+stack: preview 9.2
+serverless: preview
+```
+
+In **Discover**, LOOKUP JOIN commands include interactive options that let you create or edit lookup indices directly from the editor.
+
+#### Create a lookup index from the editor
+
+You can create a lookup index directly from the ES|QL editor. To populate this index, you can type in data manually or upload a CSV file up to 500 MB.
+
+To create lookup indices, you need the [`create_index`](elasticsearch://reference/elasticsearch/security-privileges.md#privileges-list-indices) {{es}} privilege on the corresponding pattern.
+
+1. In your {{esql}} query, add a `LOOKUP JOIN` command. For example:
+   ```esql
+   FROM kibana_sample_data_logs
+   | LOOKUP JOIN
+   ```
+   Add a space after the command. The editor suggests existing lookup indices and offers to create one. You can also type an index name in your query. If it doesn't exist, the editor suggests to create it.
```

Comment:
I'd like one of your trademark `.gif`s here now 💅 😄

Reply:
🤣🫡

Reply:
I don't think it's worth it, es|ql users will be familiar with how the autocomplete works by the time they start doing JOINs in their queries, and the crud UI is fairly straightforward.